azure-pipelines-ui-tests.dfe.yml

```diff
@@ -1,11 +1,30 @@
 variables:
   SPN_NAME: s101d-datahub-spn-ees-dfe-gov-uk
 schedules:
-- cron: 0 6 * * 1,2,3,4,5
+- cron: 0 6 * * 1,2,3,4
   branches:
     include:
     - dev
   always: true
+
+- cron: 0 6,14,18,23 * * 5
+  branches:
+    include:
+    - dev
+  always: true
+
+- cron: 0 6,14,18,23 * * 6
+  branches:
+    include:
+    - dev
+  always: true
+
+- cron: 0 6,14,18,23 * * 0
```

Comment:
I'm inclined to think that running them 4 times on saturday and sunday is excessive, especially as the tests rerun failed test suites themselves.

Maybe change this so they run just a couple of times, say once on saturday night and once on sunday night? I think that would be enough to reveal flaky test issues.

So I'd be inclined to go for one new cron string of `0 23 * * 0,6`

Reply:
Discussed this with Sai. We're adding these extra runs as a temporary measure just ascertain how flaky the tests are - so given that we're just doing this for one weekend, I'm happy for this to be approved.